src/particles.zig

```diff
@@ -60,12 +65,22 @@ pub const ParticleManager = struct {
 		};
 
 		const particleType = readTextureDataAndParticleType(assetsFolder, textureId);
+		var rotVel: RandomRange(f32) = RandomRange(f32).fromZon("rotationVelocity", zon) orelse .init(20, 60);
```

Comment:
```suggestion
		var rotVel: RandomRange(f32) = .fromZon("rotationVelocity", zon) orelse .init(20, 60);
```

Reply:
Ah I see, I thought this would work. I made an upstream issue for this: https://codeberg.org/ziglang/zig/issues/30639